pkg/apis/networkaddonsoperator/shared/networkaddonsconfig_types.go

```diff
@@ -77,14 +77,3 @@ type Container struct {
 	Name       string `json:"name"`
 	Image      string `json:"image"`
 }
-
```

Comment:
why is this shared.NetworkAddonsConfig removed?
why do we move it to `networkaddonsconfig_controller.go`?

Reply:
If we have it at pkg/apis the gen crds will genreate a "shared" version in the yaml